lib/loadtest.js

```diff
@@ -304,6 +304,7 @@ function HttpClient(operation, params)
 				var result = {
 					statusCode: connection.statusCode,
 					body: body,
+					headers: connection.headers
```

Comment:
Nice! Published as 1.4.3.

Alex.

On Fri, Jun 10, 2016 at 1:17 PM, venetahristova <notifications@github.com>
wrote:

> In lib/loadtest.js
> <https://github.com/alexfernandez/loadtest/pull/104#discussion_r66597819>:
>
> > @@ -304,6 +304,7 @@ function HttpClient(operation, params)
> >  				var result = {
> >  					statusCode: connection.statusCode,
> >  					body: body,
> > +					headers: connection.headers
>
> done :)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/alexfernandez/loadtest/pull/104/files/a70f682407c2b8a5ea10d92decfe9679a5529f5e#r66597819>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA1gGnwaFlNjNpBPULrTMcL4s3gl5F3Pks5qKUeygaJpZM4Iy1NL>
> .
>